manifests/init.pp

```diff
@@ -258,16 +258,16 @@
 #
 # $server_manage_user::                     Whether to manage the server user resource
 #
-# $server_user::                            Name of the puppetmaster user.
+# $server_user::                            Name of the puppetserver user.
```

Comment:
Not sure I like the description. It never really was that good to begin with. Perhaps something like:
```suggestion
# $server_user::                            Username used for the puppetserver  process
```

Then similar for the group.